### PR TITLE
Feature/reorganise repo

### PR DESCRIPTION
**Pull request by rossmiller**
This amends the folder structure, config and approach to Django setup and docker to bring the project into line with other Live Services projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,99 +4,39 @@
...
-a fully dockerised environment containerised and integrated together.
-To use it, clone the repository at the same level of the api, caseworker and public
-repositories and run `docker-compose-up` to bring it up.
-More information is within the repository.
-
-
-## Deployment
-
-Trade Remedies Case Worker UI configuration is performed via the following environment variables:
+Firstly, you should copy local.env.example to local.env and add the necessary environment variables for a local development environment.  local.env is in .gitignore and should not be committed to the repo.
```
Comment (jwgulley) [inline README.md:18]: Is it the case that we will no longer be able to run standalone? i.e. without docker?  If so, presumably we don't foresee any circumstances where that might be helpful e.g. to eliminate docker from our enquries if there's a problem?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
+version: "3"
 services:
-  cli:
+  caseworker:
     build: .
-    stdin_open: true
-    tty: true
     ports:
-      - "8001:8001"
+      - "8001:8000"
```
Comment (jwgulley) [inline docker-compose.yml:6]: Is that a port range?  Just that I am used to seeing a range in ascending order.

**Review by jwgulley — COMMENTED** [review]
Looks good to me, just a couple of questions and some of the comments in the other PRs also apply here.  Please drop me a line if you would like me to test this and other PRs on my PC.

File: Dockerfile [hunk 3]
```diff
@@ -1,20 +1,7 @@
...
-ENV DOCKERIZE_VERSION v0.6.1
-RUN wget -q https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
-
-EXPOSE 8001
-
-ENTRYPOINT ./start.sh
+RUN pip install --upgrade pip
+RUN pip install -r requirements/dev.txt
+RUN pip install /app/trade_remedies_client/trade-remedies-client-0.1.tar.gz
```
[the remark this answers is not in the document]
Comment (rossmiller) [reply Dockerfile:7]: No, it's exposed in the docker-compose file.

File: README.md (hunk 1 above)
Comment (rossmiller) [reply README.md:18]: No, I can't currently foresee any circumstances where we want to remove docker.

File: docker-compose.yml (hunk 2 above)
Comment (rossmiller) [reply docker-compose.yml:6]: It's the mapping between the container and host port - host:container

**Review by markhigham — APPROVED** [review]

**Pull request merged**